eudemo/eudemo/reactor.py

```diff
@@ -329,13 +329,13 @@ def _read_json(file_path: str) -> Dict:
         reactor.tf_coils.boundary(),
     )
 
-    reactor.coil_structures = build_coil_structures(
-        params,
-        build_config.get("Coil structures", {}),
-        tf_coil_xz_face=reactor.tf_coils.xz_face(),
-        pf_coil_xz_wires=reactor.pf_coils.PF_xz_boundary(),
-        pf_coil_keep_out_zones=[upper_port_xz],
-    )
+    # reactor.coil_structures = build_coil_structures(
+    #     params,
+    #     build_config.get("Coil structures", {}),
+    #     tf_coil_xz_face=reactor.tf_coils.xz_face(),
+    #     pf_coil_xz_wires=reactor.pf_coils.PF_xz_boundary(),
+    #     pf_coil_keep_out_zones=[upper_port_xz],
+    # )
```

Comment:
Should we be committing this commented code? If so, we should add a comment justifying why we've commented it out

Reply:
This has to be commented out for now as coil structures aren't working properly when built.
Can uncomment, but you pretty ugly CAD.